djstripe/migrations/0014_alter_transfer_destination.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.2.4 on 2021-06-22 10:44
```

Comment:
Can you avoid introducing multiple migrations per PR, especially per commit, please. In the end the release will only have a single migration but it makes it tougher to review the changes in question.

Reply:
I split the replacement of `StripeIdField` to `StripeForeignKey` over multiple migrations on purpose so that it is easy to navigate through the changes and to also revert them if need be. I was under the impression, this is considered a best practice to split changing a `CharField` type to a `ForeignKey` field type over multiple files. It should be possible to `squash` them into 1 migration file. Let me know.

I'll definitely make sure to `squashmigrations` from now on.

Reply:
OK, if it's on purpose then that is fine yes. But then I need to re-review the migrations themselves because I think I might misunderstand what you're trying to do with them.

Reply:
@jleclanche I think we will encounter the same issue with migrations every time a `StripeIdField` is converted to a `StripeForeignKey`. One solution I was thinking of was perhaps modifying `_stripe_object_field_to_foreign_key` or adding another method similar to it that would recursively `fetch` and `sync` all relationships by also following `StripeIdField`s in addition to `StripeForeignKey`s. This would serve as a permanent solution for all existing `StripeIDfield`s but that may lead to a host of other problems. 

What do you think?

Reply:
I don't think that's a viable solution, no. In general we shouldnt really have StripeIDFields, they should all be foreign keys.

Reply:
Yeah, that would be better any day.